rails-ujs/submit-form.md

```diff
@@ -41,7 +41,7 @@ Works
 
 ```js
 import Rails from "@rails/ujs";
```

Comment:
I should probably redo the whole category with @rails/requestjs 🙈

Reply:
Oh right, yeah. I missed that part

Reply:
nevermind, that's a whole different story